core/src/services/s3/compatible_services.md

```diff
@@ -1,12 +1,15 @@
-## AWS S3
+
```

Comment:
We can merge this into `docs.md`.

Reply:
Make sense.

Reply:
I think examples should be put ahead of compatible services. So I didn't merge it.